plugins/outputs/postgresql/postgresql_test.go

```diff
@@ -0,0 +1,623 @@
+package postgresql
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Please use only `require` if possible.

Reply:
Strongly disagree with this as it's not idiomatic go. Tests should output all available information and not exit early.